### ISSUE TITLE
Infinite scrolling with the dataprovider

### ISSUE THREAD

**Issue by andreabioli** [vaadin/vaadin-grid#934]
I cannot find any example on how to realize an infinite (remote) scrolling with the new dataprovider component.
I found only old examples, here and there, and the closest one provided with the grid is too simple to reach this important goal. I mean: surely it's not only me to need such a behavior...
I tried modifying the examples, but I get 'repeating' tiles (every group of rows is always the same!) and I don't know how to implement a proper 'stop' condition, when reaching the end of the dataset.
I would like a solution where I don't have to ask explicitly the size of the dataset upfront...
Thank you!

**Comment by samiheikki** [comment]
Hi! Have you taken a look at this example: https://cdn.vaadin.com/vaadin-grid/3.0.0/demo/data.html#assigning-remote-function-data

**Comment by andreabioli** [comment]
Yes, and it was just the subject of my request: you have to know in advance the size of the result, so needing one further full trip to the server just for this...

**Comment by aduchate** [comment]
You can set the size of the grid just before calling the callback. This works for me.
